hudi-spark-datasource/hudi-spark/pom.xml

```diff
@@ -274,6 +280,7 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
+      <version>${avro.version}</version>
```

Comment:
same question: do we need this? thought it inherits the same version from the root pom